scripts/zones/Eastern_Adoulin/npcs/Octavien.lua

```diff
@@ -0,0 +1,56 @@
+-----------------------------------
+-- Area: Eastern Adoulin (257)
+--  NPC: Octavien
+-- Type: Palace Guard
+-- Starts Children of the Rune
+-- !pos 100.580 -40.150 -63.830
+-----------------------------------
+require("scripts/globals/keyitems")
+require("scripts/globals/quests")
+local ID = require("scripts/zones/Eastern_Adoulin/IDs")
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+  -- CHILDREN OF THE RUNE
+  local childrenOfTheRuneQuestStatus = player:getQuestStatus(ADOULIN, tpz.quest.id.adoulin.CHILDREN_OF_THE_RUNE)
+  if ENABLE_SOA == 1 and childrenOfTheRuneQuestStatus == QUEST_AVAILABLE and player:getMainLvl() >= ADVANCED_JOB_LEVEL then
+   player:startEvent(23)
+  elseif childrenOfTheRuneQuestStatus == QUEST_ACCEPTED and player:hasKeyItem(tpz.ki.YAHSE_WILDFLOWER_PETAL) then
+    player:startEvent(26)
+  elseif player:getCharVar("ChildrenOfTheRuneRewardPending") == 1 then
+    player:startEvent(29)
+  else
+    player:startEvent(27) -- Standard dialog
```

Comment:
I couldn't verify this. If anyone has retail and can check the default message, that would be great.

Went through his chat options and I believe this is most likely to be the default.